### PR TITLE
Remove email verification on project verification form

### PR DESCRIPTION
**Pull request by kkatusic**
- https://github.com/Giveth/giveth-dapps-v2/issues/4579

## Summary by CodeRabbit

- **New Features**
	- Enhanced project verification forms with email-related fields for improved tracking and management.
	- Added optional reviewer assignment during status updates for increased flexibility.
	- Introduced timestamps for verification processes to ensure accurate record-keeping.

- **Bug Fixes**
	- Improved error handling for project verification forms to provide localized messages when forms are not found.

### CODE REVIEW

**Review by CarlosQ96 — APPROVED** [review]
Fix the comment because jajajaja but all good!

File: src/repositories/projectVerificationRepository.ts [hunk 1]
```diff
@@ -27,6 +27,10 @@ export const createProjectVerificationForm = async (params: {
   return ProjectVerificationForm.create({
     project,
     user,
+    // This has been added becasue we are now doing verification of the email on user profile
+    email: user?.email || '',
```
[the remark this answers is not in the document]
Comment (kkatusic) [reply src/repositories/projectVerificationRepository.ts]: added like that

**Comment by kkatusic** [comment]
> Fix the comment because jajajaja but all good!

Done, thx @CarlosQ96

**Review by CarlosQ96 — APPROVED** [review]
Thanks!

**Pull request merged**